core/layout.go

```diff
@@ -295,13 +295,13 @@ func (ls *geomState) contentRangeDim(d math32.Dims) (cmin, cmax float32) {
 // totalRect returns Pos.Total -- Size.Actual.Total
 // as an image.Rectangle, e.g., for bounding box
 func (ls *geomState) totalRect() image.Rectangle {
-	return math32.RectFromPosSizeMax(ls.Pos.Total, ls.Size.Actual.Total)
```

Comment:
We had the previous behavior for a reason; I think it might have been related to border or shadow rendering or something. See https://github.com/cogentcore/core/commit/c3ca7fa41d79826f21612532fa7524432e35bee8; you must test very thoroughly.

Reply:
This was the commit I was thinking of where we previously switched to using actual in another context to fix #1012: 316494b28b932079d29ccaba87406a1a2ef54323